cassandane/Cassandane/Cyrus/Master.pm

```diff
@@ -531,6 +531,10 @@ sub test_service_exit_during_start
     $self->assert_null($lemm);
     $self->assert_deep_equals({ A => { live => 0, dead => 5 } },
                               $self->lemming_census());
+
+    xlog $self, "check that the error was syslogged";
+    my @loglines = $self->{instance}->getsyslog();
+    $self->assert(grep { m/too many failures for service/ } @loglines);
```

Comment:
This will give false failures if the syslog replacement injection had failed.  Have a look at `$self->assert_syslog_matches()`... I can't remember if it's exactly suitable here, but if it's not, at least the implementation will show the right way to handle this sort of thing.

Reply:
yes, can probably just use that :)  Thanks

Reply:
I added this!

We can test new behaviour by hand by killing something every second and watching the logs